inc/issue.class.php

```diff
@@ -77,6 +77,36 @@ public static function install(Migration $migration) {
     * @see CommonGLPI::display()
     */
    public function display($options = array()) {
+      $itemtype = $options['sub_itemtype'];
+      if (!in_array($itemtype, array('Ticket', 'PluginFormcreatorFormanswer'))) {
+         html::displayRightError();
+      }
+      //if (plugin_formcreator_replaceHelpdesk() == PluginFormcreatorEntityconfig::CONFIG_SIMPLIFIED_SERVICE_CATALOG) {
```

Comment:
If a user forges an URL to display an other subtype, I prefer to ensure this will not be allowed.